addons/binding/org.openhab.binding.upb/src/main/java/org/openhab/binding/upb/handler/UPBBridgeHandler.java

```diff
@@ -0,0 +1,192 @@
+/**
+ * Copyright (c) 2010-2016, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.upb.handler;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+import org.apache.commons.lang.StringUtils;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.openhab.binding.upb.internal.MessageBuilder;
+import org.openhab.binding.upb.internal.UPBMessage;
+import org.openhab.binding.upb.internal.UPBReader;
+import org.openhab.binding.upb.internal.UPBWriter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import gnu.io.CommPortIdentifier;
+import gnu.io.NoSuchPortException;
+import gnu.io.PortInUseException;
+import gnu.io.SerialPort;
+import gnu.io.UnsupportedCommOperationException;
+
+/**
+ * Handler for Universal Powerline Bus (UPB) that reads and writes messages to
+ * and from the UPB modem.
+ *
+ * @author Chris Van Orman
+ * @since 1.9.0
+ */
+public class UPBBridgeHandler extends BaseBridgeHandler implements UPBMessageSender, UPBReader.Listener {
+
+    private static final String CONFIG_NETWORK = "network";
+    private static final String CONFIG_PORT = "serialPort";
+
+    private static final Logger logger = LoggerFactory.getLogger(UPBBridgeHandler.class);
+
+    private String port;
+    private byte network = 0;
```

Comment:
I believe this is the default value for this type, any reason you initialize it explicitly?